gateway/cli/src/main.rs

```diff
@@ -96,6 +102,51 @@ pub enum Commands {
         #[clap(long)]
         network: Option<bitcoin::Network>,
     },
+    #[command(subcommand)]
+    Lightning(LightningCommands),
+}
+
+/// This API is intentionally kept very minimal, as its main purpose is to
+/// provide a simple and consistent way to establish liquidity between gateways
+/// in a test environment.
+#[derive(Subcommand)]
+pub enum LightningCommands {
+    /// Connect to another lightning node
+    ConnectToPeer {
+        #[clap(long)]
+        pubkey: bitcoin::secp256k1::PublicKey,
+
+        #[clap(long)]
+        host: String,
+    },
+    /// Get a Bitcoin address to fund the gateway
+    GetFundingAddress,
+    /// Open a channel with another lightning node
+    OpenChannel {
+        /// The public key of the node to open a channel with
+        #[clap(long)]
+        pubkey: String,
+        /// The amount to fund the channel with
+        #[clap(long)]
+        channel_size_sats: u64,
+        /// The amount to push to the other side of the channel
+        #[clap(long)]
+        push_amount_sats: Option<u64>,
+    },
+    /// Wait for the lightning node to be synced with the blockchain
+    WaitForChainSync {
+        /// The block height to wait for
+        #[clap(long)]
+        block_height: u32,
+
+        /// The maximum number of retries
+        #[clap(long)]
+        max_retries: Option<u32>,
+
+        /// The delay between retries
+        #[clap(long)]
+        retry_delay_seconds: Option<u64>,
```

Comment:
Leaving this for now while we figure out if we're even going to keep `WaitForChainSync`

Reply:
Update: As discussed elsewhere, the under-the-hood behavior has been refactored so that rather than having implementation-specific wait-for-chain-sync functions that each have their own polling logic, the CLI command polls `gateway info` and checks the returned block count until it matches or exceeds the requested height.